spec/jobs/remove_vacancies_that_expired_yesterday_from_google_index_job_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe RemoveVacanciesThatExpiredYesterdayFromGoogleIndexJob do
+  let(:urls) { Vacancy.expired_yesterday.map { |vacancy| Rails.application.routes.url_helpers.job_url(vacancy) } }
+
+  before do
+    create_list(:vacancy, 5, expires_at: 1.day.ago)
+    create_list(:vacancy, 1, expires_at: 2.days.ago)
```

Comment:
You created this vacancies that expired on a day that wasn't yesterday (and a vacancy that isn't expired), but you never checked that RemoveGoogleIndexQueueJob didn't run on them, so I don't know what the point of creating them was.

Reply:
Have added another test for this.